### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SehyunPark**
specifying what IRC stands for

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,9 +129,9 @@ Check out our FAQs and get answers to common questions:
 
 + https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions
 
-Talk to us on IRC:
+Talk to us on IRC(Internet Relay Chat):
```
Comment (Snuffleupagus) [inline README.md]: There's a space missing between `IRC` and `(Internet Relay Chat)`.

File: README.md [hunk 2]
```diff
@@ -129,9 +129,9 @@ Check out our FAQs and get answers to common questions:
...
 
-+ #pdfjs on irc.mozilla.org
++ #pdfjs on irc.mozilla.org
```
Comment (Snuffleupagus) [inline README.md]: Please remove the added trailing space here.

File: README.md (hunk 1 above)
Comment (SehyunPark) [reply README.md]: edited as requested

File: README.md (hunk 2 above)
Comment (SehyunPark) [reply README.md]: edited as requested

**Comment by Snuffleupagus** [comment]
Please squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits (which is linked from https://github.com/mozilla/pdf.js/wiki/Contributing#3-make-changes).

**Comment by SehyunPark** [comment]
Thanks for the reply and your kind explanation on how to squash and merge
but I clicked merge button (with allowing merge & squash), and I can't find squash button for squashing two commits through desktop github. Would you please squash the two commits? Or am I the only person who could do squash?

**Comment by Snuffleupagus** [comment]
If you please follow the workflow outlined in https://github.com/mozilla/pdf.js/wiki/Contributing, rather than editing files directly on GitHub (which only has limited features), then squashing shouldn't be a problem.

**Comment by SehyunPark** [comment]
closed since using another method (to use Linux browser, not in Github desktop)

**Pull request closed without merging**